OrcScala/src/orc/run/extensions/SupportForSiteInvocation.scala

```diff
@@ -17,23 +17,81 @@ import orc.Handle
 import orc.values.sites.Site
 import orc.error.OrcException
 import orc.error.runtime.JavaException
+import orc.values.sites.DirectSite
+import orc.values.Field
+import orc.Invoker
+import orc.DirectInvoker
+import orc.error.runtime.ExceptionHaltException
+import orc.error.runtime.HaltException
+import orc.values.sites.InvokerMethod
+import orc.error.runtime.UncallableValueException
+import orc.error.runtime.NoSuchMemberException
+import orc.error.runtime.DoesNotHaveMembersException
+import orc.values.sites.AccessorValue
+import orc.ErrorInvoker
+import orc.ErrorAccessor
+
+// TODO: Implement Invoker away InvocationBehavior. One for classes that are Invokers, one for an invoker creation interface. Similarly for Accessor. 
```

Comment:
That's because it doesn't make any sense. I don't even know what I was thinking when I typed this. However I think I already did when I was refering to in the later sentence fragments. I'll check and remove this.